velodyne_pointcloud/include/velodyne_pointcloud/datacontainerbase.h

```diff
@@ -127,6 +124,14 @@ class DataContainerBase
     {
       cloud.header.frame_id = config_.target_frame;
     }
+    else if (!config_.fixed_frame.empty())
+    {
+      cloud.header.frame_id = config_.fixed_frame;
+    }
+    else
+    {
+      cloud.header.frame_id = sensor_frame;
+    }
```

Comment:
It should be made clear somewhere (wiki.ros.org?) that fixed_frame > target_frame > sensor_frame.

Reply:
Yes that makes sense to document this properly. I would describe it in a similar way as I did in the bullet points of [this comment](https://github.com/ros-drivers/velodyne/issues/342#issuecomment-633029124) when I described the behavior in my new version.

Reply:
> fixed_frame > target_frame > sensor_frame

Not sure what you exactly mean by this. Do you mean the parameter precedence?
Then it should be:
`target_frame > fixed_frame > sensor_frame`

Reply:
Yes, sorry. I got the order wrong.